packages/fxa-settings/src/models/Account.ts

```diff
@@ -530,6 +530,61 @@ export class Account implements AccountData {
     }
   }
 
+  async verifyPasswordForgotToken(token: string, code: string) {
```

Comment:
For FxA, we need two tokens to reset a users password. The `passwordForgotToken` is what is sent via email (along with a code). After a user clicks the reset link from their email, this token is then exchanged for an `accountResetToken`.

Reply:
This could be useful as a comment in code 👀